### DISCUSSION TITLE
An option to always start a profile with 5s full speed at first (pump the air out)

### DISCUSSION THREAD

**Discussion by UrmasK222** [Rem0o/FanControl.Releases#914] [Ideas]
Hi

Usually when I start playing a game I choose full speed profile at first for 5-10s (AIO pumps, fans) so that the air bubbles will get pushed out from the pump. I can hear the positive effect of this action almost always.

It would be great if I had an option where every time, when I start Fan Control, it's done automatically. :)

br
u

**Comment by Migratingcocofruit** [comment]
There shouldn't be any bubbles in your pump even when your computer is turned off, and pumping them through will likely shorten your pump's lifespan. 
Make sure your pump isn't the highest point in the loop, ideally that would be the reservoir part of your radiator, and if mounting the radiator vertically, run the tubes from the pump towards the bottom rather than the top.
